web/src/admin.js

```diff
@@ -269,6 +271,10 @@ export function build_page() {
 
         tippy($("#realm_can_access_all_users_group_widget_container")[0], opts);
     }
+
+    if (options.realm_direct_message_permission_group_id === 9) {
```

Comment:
Never use hardcoded values like 9 here. For one thing, this value happens to be 9 for the data in the development environment. But even if this were an enum with policy names, you should always use a semantic name so that one can understand what is happening; see `settings_data.ts` and `settings_config.ts` for some patterns.

Check how other `_group_id` settings work.

Reply:
Oh! I was going to change that before pushing the changes but forgot to do that. I changed that now.